### PR TITLE
Finished code-challenges, coding-concepts, and interview-questions

### PR DESCRIPTION
**Pull request by jhan-dev**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ ✅] Coding Challenges
- [ ✅] Coding Concepts
- [ ✅] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -7,62 +7,120 @@
...
+// }
+
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
+    //First create an if else statement to check if const temp is below 212, above 212, or = to 212.
 
-const temp = 42
+    //If below 212, console log `${temp} is below boiling point`
+    //else if above 212, console log `${temp} is above boiling point`
```
Comment (elysemontano) [inline code-challenges.js:44]: I see in your pseudocode to use an else if statement for the other statements, but I only see if statements below.  I think you were heading in the right direction with your pseudocode as else if will allow you to run any of these conditions at a given time.

File: coding-concepts.js [hunk 2]
```diff
@@ -9,26 +9,26 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:  5
```
Comment (elysemontano) [inline coding-concepts.js:12]: Great work in this section!

File: interview-questions.md [hunk 3]
```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
...
-Your answer:
+Your answer:    Git is a version control software. Github is the cloud server version of a version control software.
 
-Researched answer:
+Researched answer:  Git is a open source version control software while Github is a web app service that hosts the majority of repositories with collab services and tools.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer:    && and ||
```
Comment (elysemontano) [inline interview-questions.md:23]: When answering interview questions, you should expand using definitions and examples when possible. One-word or two-word answers in an interview setting will not show your understanding and knowledge, and will likely come off as uninterested.

**Review by elysemontano — APPROVED** [review]
Great work Joseph!  Your logic for the code problems were great.  I would like to see you expand more with your explanations, especially on the interview questions section in future assessments.

File: code-challenges.js (hunk 1 above)
Comment (jhan-dev) [reply code-challenges.js:44]: Understood! I'll try focusing more on the interview portion with better explainations from now on. Thanks for the feedback Elyse.

**Pull request merged**